src/integration-test/tests/citizen/defence/steps/statementOfMeans.ts

```diff
@@ -25,22 +26,25 @@ const somDebtsPage: DebtsPage = new DebtsPage()
 const somMonthlyIncomePage: MonthlyIncomePage = new MonthlyIncomePage()
 const somMonthlyExpensesPage: MonthlyExpensesPage = new MonthlyExpensesPage()
 const somCourtOrdersPage: CourtOrdersPage = new CourtOrdersPage()
+const somCannotPayImmediatelyPage: CannotPayImmediatelyPage = new CannotPayImmediatelyPage()
 
 export class StatementOfMeansSteps {
 
   fillStatementOfMeansData (): void {
     somStartPage.clickContinue()
+    somBankAccountsPage.clickContinue()
+    somBankAccountsPage.clickContinue()
```

Comment:
Why it happens twice?

Reply:
Typo error will remove. Bank account page moved to the front of the journey